test/07-openqa-label-known-issues.t

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+source test/init
+bpan:source bashplus +err +fs +sym
+
+plan tests 15
+
+source openqa-label-known-issues
+client_args=(api --host "$host_url")
+
+export KEEP_REPORT_FILE=1
+
+client_output=''
+mock-client-output() {
+    client_output+="client_call $@"$'\n'
+    echo "$client_output\n"
+}
+
+client_call=(mock-client-output "${client_call[@]}")
+autoinst_log=$dir/data/04-autoinst.txt
+
+try-client-output() {
+    client_output=''
+    try "$*"' && echo "$client_output"'
+}
+openqa-cli() {
+    local id=$(basename "$4")
+    cat "$dir/data/$id.json"
+}
+curl() {
+    if [[ "$7" =~ /(404|414|101|102)/file/autoinst-log.txt ]]; then
+        echo -n "404"
+    else
+        echo -n "200"
+    fi
+}
+comment_on_job() {
+    local id=$1 comment=$2 force_result=${3:-''}
+    echo "$comment"
+}
+out=''
+hxnormalize() {
+    echo -n "$out";
+}
+hxselect() {
+    echo -n "$out";
+}
```

Comment:
This fix works for me in github actions:
```suggestion
hxnormalize() {
    cat "$out"
}
hxselect() {
    cat "$out"
}
```

Reply:
Is that supposed to mock the `if hxnormalize -x "$out" | hxselect -s '\n' -c '` line?

If yes, the mocked `hxnormalize` could just output `$2` (unless you need some fancy features) and `hxselect` its input as it clearly works as a filter... although I am not exactly sure what is the intended goal.

```suggestion
hxnormalize() {
    echo -n "$2"
}
hxselect() {
    cat -
}
```

Reply:
@baierjan u nail it. I just apply it but I would go back to read the comment to understand it a bit better. but it is very nice to see it working.

Reply:
and I see how silly I was if I had `;` in all the attempts. thanks again

Reply:
But it has been resolved.